docs/operation-guide.md

```diff
@@ -128,11 +134,16 @@ To create a full backup job, modify `fullbackup` section in `values.yaml` file.
 * `schedule` takes the [Cron](https://en.wikipedia.org/wiki/Cron) format
 * `user` and `password` must be set to the correct user which has the permission to read the database to be backuped.
 
-If TiDB cluster is running on GKE, the backup data can be uploaded to GCS bucket. A bucket name and base64 encoded service account credential that has bucket read/write access must be provided. The comments in `values.yaml` is self-explanatory for GCP backup.
+```shell
+$ releaseName="tidb-backup"
+$ namespace="tidb"
+$ helm install charts/tidb-backup --name=${releaseName} --namespace=${namespace}
+$ watch kubectl get po -n ${namespace} -l app.kubernetes.io/component=fullbackup -o wide
+```
 
-### Incremental backup
+> **Note:** You must set the fullbackup PV's [reclaim policy](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy) to `Retain` to safe your backup data.
```

Comment:
safe -> save

Reply:
here, I mean to keep the data **safe**, the pv is retained when the pvc is deleted.

Reply:
oh, then it would be "to keep your backup data safe".